test/cuda/Makefile

```diff
@@ -0,0 +1,16 @@
+include $(HCPP_ROOT)/include/hcpp.mak
+
+TARGETS=allocate copy kernel
+
+FLAGS=-g -std=c++11
```

Comment:
This is for a particular example. We wouldn't want it to apply to all hclib projects, everywhere.